build/bootstrap.sh

```diff
@@ -0,0 +1,220 @@
+#! /bin/bash
+
+###
+# This script handles provisioning of the Delivery Optimization client components on supported platforms
+###
+
+# bootstrap scripts will exit immediately if a command exits with a non-zero status
+set -e
+
+# Defaults 
+INSTALL=all
+
+usage() {
+    cat <<EOM
+$(basename $0) - Script to setup development environments for Delivery Optimization
+Usage: $(basename $0) --platform <platform to install for> --install <install command>
+    --platform          # Platform to provision, supported platforms: Ubuntu1804, Ubuntu 2004, Debian9, Debian 10. Default is Ubuntu1804
+    --install           # Which command to run, supported commands: builddependencies, developertools, containertools, qemu, all. Default is All
+EOM
+    exit 1
+}
+
+function parseArgs() {
```

Comment:
>function parseArgs() [](http://example.com/codeflow?start=0&length=20)

not the most familiar with shell scripting, but I think the () can be removed here, usage() can also be a function to stay consistent